docs/modules/blocks/enderchests.mdx

```diff
@@ -0,0 +1,200 @@
+---
+id: enderchests
+title: Ender Chests
+---
+
+This module allow mapmakers to have better control over Ender chest behaviors in a map.
+
+The `<enderchest>` tag can contain any number of `<dropoff>` tags. Each `<dropoff>` can use a region and a filter to
+determine where and when items should be dropped from the Ender chest.
```

Comment:
Elaborate on what the drop offs do. The ender chests are like vanilla, each player has a "profile" on the chest. The drop off determines where their items will drop when the player leaves or switches teams. Fall back is for when a drop off isn't possible.